modules/lobby/client/config/lobby.client.config.js

```diff
@@ -2,7 +2,14 @@
 
 // Configuring the Articles module
 angular.module('lobby').run(['Menus',
-  function (Menus) {}
+  function (Menus) {
+    Menus.addMenuItem('topbar', {
+      title: 'Lobby',
+      state: 'home',
+      position: 1,
+      roles: ['*']
+    });
+  }
```

Comment:
Adds a new menu for the Topic creation/update UI. There's nothing there yet though - maybe I should remove this from this PR?